### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by door5719** [AlexeyAB/darknet#1176]
Hi,i want to know how to change the loss function,just as box , conf and class .I want to train tiny-yolo model with other datas(such as from yolov3) but not the ground truth,just like knowledge distillation learning style. The idear is from  paper(Object detection at 200 Frames Per Second) i got from  https://arxiv.org/pdf/1805.06361.pdf.  can you give same advice? Thank you ~~
